app/src/main/java/org/covidwatch/android/data/backend/InfectionState.kt

```diff
@@ -0,0 +1,9 @@
+package org.covidwatch.android.data.backend
+
+/**
+ * Enumeration of all possible infection states of a person
+ */
+enum class InfectionState(val isPotentiallyInfectious: Boolean) {
```

Comment:
I like it, interested to see how we tie this forward in the rest of the app.

Reply:
Yes, enums are much more speaking than bools. But I wanted for now not too many conflicts, so I did not propagate this too far on in the app.